src/Paysera/Bundle/CodeGeneratorBundle/Service/PropertyDefinitionBuilder.php

```diff
@@ -15,12 +15,7 @@ class PropertyDefinitionBuilder
         'time-only',
     ];
 
-    /**
-     * @param string $name
-     * @param array $definition
-     * @return PropertyDefinition
-     */
-    public function buildPropertyDefinition($name, array $definition)
+    public function buildPropertyDefinition(string $name, array $definition)
```

Comment:
could be `public function buildPropertyDefinition(string $name, array $definition) : PropertyDefinition`